reactive-messaging-kafka: CNFE: io.quarkus.netty.runtime.NettyTemplate

**Describe the bug**
A project with `quarkus-smallrye-reactive-messaging-kafka` and `quarkus-smallrye-reactive-messaging` works fine in the `quarkus:dev` mode but fails to start from jar.
Also fails to build to a native image.

**Expected behavior**
The application should start without error and it should be possible to build a native image.
Adding a dependency on `quarkus-vertx` solves the problem.
I think it may be that the dependency on `quarkus-vertx` is missing from  `quarkus-smallrye-reactive-messaging-kafka`. There is a dependency between the deployment modules.

**Actual behavior**
The following exception is thrown when the jar is executed:
```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at io.quarkus.runner.GeneratedMain.main(Unknown Source)
Caused by: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.runner.ApplicationImpl1.<clinit>(Unknown Source)
        ... 1 more
Caused by: java.lang.NoClassDefFoundError: io/quarkus/netty/runtime/NettyTemplate
        at io.quarkus.deployment.steps.NettyProcessor$createExecutors2.deploy(Unknown Source)
        ... 2 more
Caused by: java.lang.ClassNotFoundException: io.quarkus.netty.runtime.NettyTemplate
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 3 more
```


**To Reproduce**
Steps to reproduce the behavior:
1. generate a project
2. add quarkus-smallrye-reactive-messaging-kafka
3. add quarkus-smallrye-reactive-messaging
4. build with `mvn clean package`
5. execute with java -jar target/<my-app>-runner.jar

**Configuration**
```properties
# Add your application.properties here, if applicable.
smallrye.messaging.sink.queries.type=io.smallrye.reactive.messaging.kafka.Kafka
smallrye.messaging.sink.queries.topic=queries
smallrye.messaging.sink.queries.bootstrap.servers=localhost:9092
smallrye.messaging.sink.queries.key.serializer=org.apache.kafka.common.serialization.StringSerializer
smallrye.messaging.sink.queries.value.serializer=org.apache.kafka.common.serialization.StringSerializer
smallrye.messaging.sink.queries.acks=1
```


**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Linux t580 5.0.16-200.fc29.x86_64 #1 SMP Tue May 14 18:27:35 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
 - Output of `java -version`: Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
 - GraalVM version (if different from Java): 1.0.0-rc15
 - Quarkus version or git rev: master

--- a/extensions/smallrye-reactive-messaging-kafka/runtime/pom.xml
+++ b/extensions/smallrye-reactive-messaging-kafka/runtime/pom.xml
@@ -22,6 +22,10 @@
             <groupId>io.smallrye.reactive</groupId>
             <artifactId>smallrye-reactive-messaging-kafka</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
 
         <dependency>
             <groupId>org.apache.kafka</groupId>

--- a/extensions/smallrye-reactive-messaging/runtime/pom.xml
+++ b/extensions/smallrye-reactive-messaging/runtime/pom.xml
@@ -23,6 +23,10 @@
          <artifactId>quarkus-smallrye-reactive-streams-operators</artifactId>
       </dependency>
       <dependency>
+         <groupId>io.quarkus</groupId>
+         <artifactId>quarkus-vertx</artifactId>
+      </dependency>
+      <dependency>
          <groupId>io.smallrye.reactive</groupId>
          <artifactId>smallrye-reactive-messaging-provider</artifactId>
          <exclusions>

